test/functional/unified-spec-runner/runner.ts

```diff
@@ -15,6 +16,25 @@ interface MongoDBMochaTestContext extends Mocha.Context {
   configuration: TestConfiguration;
 }
 
+export function trace(message: string): void {
+  if (process.env.UTR_TRACE) {
+    console.log(` > ${message}`);
```

Comment:
Can we change this to `console.error`? even though its for debugging its important to not interfere with the mocha reporter's output.

Reply:
Updated to use `console.error`